ui/plugins/controller.js

```diff
@@ -92,7 +92,9 @@ treeherder.controller('PluginCtrl', [
             // it if job was successful.
             $http.get(thServiceDomain + '/api/project/' + $scope.repoName +
                       '/performance/data/?job_id=' + job.id).then(function(response) {
-                          if (!_.isEmpty(response.data) && job.job_type_name !== "Build") {
+                          var jobType = job.job_type_name;
+                          if (!_.isEmpty(response.data) && jobType !== "Build" &&
+                              jobType !== "Nightly" && !jobType.startsWith('build-')) {
```

Comment:
BB-based Nightly builds have a job_type_name of "Nightly".
TC-based builds/nightlies are all prefixed with "build-" (like "build-linux64/opt" or "build-linux-nightly/opt".